### PR TITLE
fix #269939: order preset list to account for similar instruments on different banks

### PR DESCRIPTION
**Pull request by jthistle**
Resolves: https://musescore.org/en/node/269939

### CODE REVIEW

File: fluid/fluid.cpp [hunk 1]
```diff
@@ -584,10 +586,17 @@ void Fluid::updatePatchList()
...
             bankOffset += (banks + 1);
             }
+
+      // Order by patch number first instead of by bank first
+      for (QList<MidiPatch*> num : patchNums) {
+         for (MidiPatch* p : num)
+            patches.append(p);
+         }
+
       /* try to set the correct presets */
```
Comment (Jojo-Schmitz) [inline fluid/fluid.cpp]: indentation issue, should be 6 spaces per level

Comment (jthistle) [reply fluid/fluid.cpp]: Not in fluid it isn't, they're using a different number.

Comment (Jojo-Schmitz) [reply fluid/fluid.cpp]: seems the surrounding code is using something different

Comment (jthistle) [reply fluid/fluid.cpp]: Oh, sorry, my mistake. Will fix.

Comment (jthistle) [reply fluid/fluid.cpp]: Fixed now.

**Pull request merged**